doc/specs/preview-monaco.md

```diff
@@ -0,0 +1,63 @@
+# **Preview pane - Adding previews with monaco**
+
+<img align="right" src="../images/overview/PT_small.png" />
+
+- **What is it:** Implementing [#1527](https://github.com/microsoft/PowerToys/issues/1527)
+- **Authors:** Aaron Junker ([@aaron-junker](https://github.com/aaron-junker)), Clint Rutkas ([@crutkas](https://github.com/crutkas))
+- **Spec Status:** Waiting for review
+
+# 1 Overview
+
+Creating a new preview handler for all Developer files. For this we use [Microsoft Monaco editor](https://github.com/microsoft/monaco-editor). 
+
+## 1.1 Technical implementation
+
+A WebView2 window implements shows HTML file with Monaco integrated.
+
+### 1.1.1 New dependencies
+
+The installer needs to install [WebView2](https://developer.microsoft.com/en-us/microsoft-edge/webview2/)
+
+### 1.1.2 Why WebView2
+
+Experiments with a POC showed that only WebView2 is displaying Monaco the right way. WebView and WebBrowser elements are using too old versions of IE/Edge who don't support JS functions like `require()`, which are needed for monaco to work. The other option would be to import many JS liabarys, but they don't guarantee 100% coverage with JS and they take more space.
```

Comment:
@sibille how does [WTS](https://github.com/Microsoft/WindowsTemplateStudio/) do this without WebView2?  It just uses a normal WebView?

Reply:
my theory is this was added in well before Monaco has this dependencies and why this is like it is

Reply:
just saw your question: Yes, we added monaco to WTS way before it had this dependency. We're using WPF and WebBrowser.